Antidote/DialingCallViewController.m

```diff
@@ -61,10 +63,30 @@ - (void)createEndCallButton
     [self.view addSubview:self.cancelCallButton];
 }
 
+- (void)createFriendAvatar
+{
+    AvatarsManager *avatars = [[AvatarsManager alloc] init];
+
+    CGFloat diameter = (1.0 / 2.0) * self.view.bounds.size.width;
+
+    OCTFriend *friend = [self.call.chat.friends firstObject];
+    UIImage *image = [avatars createAvatarFromString:friend.name diameter:diameter
```

Comment:
You should use `nickname` instead of `name`, as it is used all over the project.